workflow/usage.rst

```diff
@@ -277,3 +294,6 @@ The links below will only be displayed when the action is allowed:
     {% if workflow_has_marked_place(post, 'to_review') %}
         <p>This post is ready for review.</p>
     {% endif %}
+
+    {# Get all the places related to the 'post' marking #}
+    {{ workflow_marked_places(post)|join(',') }}
```

Comment:
This example is pretty stupid. Can you please suggest me an interesting and realistic example for `workflow_marked_places()`? Thanks!